components/authz-service/storage/postgres/migration/sql/49_add_status_field_to_rules.up.sql

```diff
@@ -0,0 +1,154 @@
+BEGIN;
+
+CREATE OR REPLACE FUNCTION
+  query_staged_and_applied_rules(_project_filter TEXT[])
+  RETURNS setof json AS $$
+
+  -- fetch staged rules
+  SELECT json_build_object(
+    'id', r.id,
+    'project_id', r.project_id,
+    'name', r.name,
+    'type', r.type,
+    'deleted', r.deleted,
+    'status', 'staged',
```

Comment:
Consider SQL constants for 'staged' and 'applied'

Reply:
Really? As far as I understand it, constants in SQL are scoped per-function. So, this functions could have a constant, but there's not much to gain, really, since both strings only occurr once.... Also it doesn't look common in our code base to have SQL constants. 😃